web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
Actually thinking about this, I think I have to use `raw`. Maybe I can scrub the description when it's entered to make sure it doesn't contain any HTML?